<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Software Reliability Growth Models Computer</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>22</y>
     <width>820</width>
     <height>602</height>
    </rect>
   </property>
   <widget class="QComboBox" name="comboBox" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>80</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>130</y>
      <width>331</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title" >
     <string>Parameters</string>
    </property>
    <widget class="QLabel" name="label_2" >
     <property name="geometry" >
      <rect>
       <x>180</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Undefined</string>
     </property>
    </widget>
    <widget class="QLabel" name="label" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text" >
      <string>Total number of faults estimate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Mean time to fault:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5" >
     <property name="geometry" >
      <rect>
       <x>180</x>
       <y>40</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Undefined</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Confidence interval:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7" >
     <property name="geometry" >
      <rect>
       <x>140</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Undefined</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8" >
     <property name="geometry" >
      <rect>
       <x>200</x>
       <y>60</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>to</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9" >
     <property name="geometry" >
      <rect>
       <x>220</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Undefined</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>160</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Errors after one month estimate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16" >
     <property name="geometry" >
      <rect>
       <x>180</x>
       <y>160</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Undefined</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>80</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Current number of faults:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18" >
     <property name="geometry" >
      <rect>
       <x>180</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Undefined</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>180</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Errors during next month:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20" >
     <property name="geometry" >
      <rect>
       <x>180</x>
       <y>180</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Undefined</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>100</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Current testing time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22" >
     <property name="geometry" >
      <rect>
       <x>180</x>
       <y>100</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Undefined</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>200</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Errors after one year estimate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24" >
     <property name="geometry" >
      <rect>
       <x>180</x>
       <y>200</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Undefined</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>220</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Errors during next year:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26" >
     <property name="geometry" >
      <rect>
       <x>180</x>
       <y>220</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Undefined</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>120</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Errors after one week estimate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28" >
     <property name="geometry" >
      <rect>
       <x>180</x>
       <y>120</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Undefined</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>140</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Errors during next week:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30" >
     <property name="geometry" >
      <rect>
       <x>180</x>
       <y>140</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Undefined</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>10</y>
      <width>581</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>Project name: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Compute</string>
    </property>
   </widget>
   <widget class="QWidget" native="1" name="graph" >
    <property name="geometry" >
     <rect>
      <x>440</x>
      <y>110</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_11" >
     <property name="geometry" >
      <rect>
       <x>230</x>
       <y>260</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Time</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line" >
    <property name="geometry" >
     <rect>
      <x>440</x>
      <y>80</y>
      <width>5</width>
      <height>311</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_10" >
    <property name="geometry" >
     <rect>
      <x>390</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text" >
     <string>Number of faults</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12" >
    <property name="geometry" >
     <rect>
      <x>430</x>
      <y>370</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13" >
    <property name="geometry" >
     <rect>
      <x>410</x>
      <y>80</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>n</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14" >
    <property name="geometry" >
     <rect>
      <x>760</x>
      <y>370</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>n</string>
    </property>
   </widget>
   <widget class="Line" name="line_2" >
    <property name="geometry" >
     <rect>
      <x>410</x>
      <y>360</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2" >
    <property name="geometry" >
     <rect>
      <x>510</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>Mean value function</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Failure intensity</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_2" >
    <property name="geometry" >
     <rect>
      <x>670</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3" >
    <property name="geometry" >
     <rect>
      <x>660</x>
      <y>430</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text" >
     <string>Compute</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>430</y>
      <width>601</width>
      <height>151</height>
     </rect>
    </property>
    <property name="rowCount" >
     <number>4</number>
    </property>
    <property name="columnCount" >
     <number>5</number>
    </property>
    <row>
     <property name="text" >
      <string>Convergence</string>
     </property>
    </row>
    <row>
     <property name="text" >
      <string>Accuracy</string>
     </property>
    </row>
    <row>
     <property name="text" >
      <string>Stability</string>
     </property>
    </row>
    <row>
     <property name="text" >
      <string>Recommended</string>
     </property>
    </row>
    <column>
     <property name="text" >
      <string>Goel-Okumoto</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Jelinski-Moranda</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Littlewood-Verrall</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>S-Shaped</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Logarithmic</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionNew_project" />
    <addaction name="actionOpen_Project" />
    <addaction name="actionDelete_project" />
    <addaction name="separator" />
    <addaction name="actionAbout" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>Edit</string>
    </property>
    <addaction name="actionChange_models_list" />
    <addaction name="actionLoad_new_XML_file" />
    <addaction name="actionSetup_data_selection" />
    <addaction name="actionAdd_data_to_current_project" />
    <addaction name="actionBatch_data_load" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>624</y>
     <width>820</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <action name="actionExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionNew_project" >
   <property name="text" >
    <string>New project</string>
   </property>
  </action>
  <action name="actionOpen_Project" >
   <property name="text" >
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionDelete_project" >
   <property name="text" >
    <string>Delete project</string>
   </property>
  </action>
  <action name="actionChange_models_list" >
   <property name="text" >
    <string>Change models list</string>
   </property>
  </action>
  <action name="actionLoad_new_XML_file" >
   <property name="text" >
    <string>Load new XML file</string>
   </property>
  </action>
  <action name="actionAdd_data_to_current_project" >
   <property name="text" >
    <string>Add data to current project</string>
   </property>
  </action>
  <action name="actionSetup_data_selection" >
   <property name="text" >
    <string>Setup data selection</string>
   </property>
  </action>
  <action name="actionBatch_data_load" >
   <property name="text" >
    <string>Batch data load</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
